examples/graphene_apps/cppopenvino/test/test_send_request.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+import subprocess
+import zmq
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+# -------------------------------------------------------------------------
+
+
+# This is a test file to send work order.
+def main(argv):
+    if len(argv) >= 1:
+        workload = argv[0]
+    else:
+        workload = None
+
+    if len(argv) >= 2:
+        params = argv[1]
+    else:
+        params = None
+
+    context = zmq.Context()
+    # TODO: read zmq url from config file.
+    zmq_url = "tcp://ov-work-order:8888"
```

Comment:
This config is already part of config, read from the config.

Reply:
No. This is only a test file which needs a different config file. I've added a TODO